contents/java/JUnit.md

```diff
@@ -0,0 +1,234 @@
+<frontmatter>
+  title: JUnit
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# JUnit
+
+<b>Author: [Lin Si Jie](https://github.com/sijie123)</b>
+
+Reviewers: [Aadyaa Maddi](https://github.com/amad-person), [Marvin Chin](https://github.com/marvinchin)
+
+## What is JUnit?
+_____
+
+JUnit is an open-source <popover effect="fade" content="An automatic test running and verfication process to determine the correctness of a system programmatically">automated testing</popover> framework for Java. Using its <popover effect="fade" content="Application Programming Interfaces">APIs</popover>, developers can write and run tests to check for correctness in their software engineering projects. 
+
+To get a better idea of what JUnit does, let's see how JUnit works with a simple example. 
+
+In this example, we need to sum up an array and return its value.
```

Comment:
Can be more concise here:

>**JUnit is an _automated testing framework_ for Java** i.e., it can be used to programmatically verify the actual behavior of Java code against the expected behavior.
>
>Let's use a simple example to see how JUnit is used. Suppose we want to test the method `calculateArraySum(int[] inputArray, int N)` which returns the sum of the first `N` values of `inputArray`.

Tweak code as follows?
inputArray -> values
N -> n